processor/file.go

```diff
@@ -186,6 +187,12 @@ DIRENTS:
 
 func newFileJob(path, name string) *FileJob {
 	extension := ""
+
+	// If there is no . in the filename or it starts with one then check if #! or other
+	if !strings.Contains(name, ".") || (name[0] == '.' && strings.Count(name, ".") == 1) {
```

Comment:
It might be marginally more efficient here to call `strings.Count` once and check for zero or one, rather than calling both `strings.Contains` and `strings.Count`.